docs/docs/command-line-flags.md

```diff
@@ -47,6 +47,7 @@ The core functionality flags can be also set by environment variable `MARATHON_O
     - "vips" can be used to enable the networking VIP integration UI.
     - "task\_killing" can be used to enable the TASK\_KILLING state in Mesos (0.28 or later)
     - "external\_volumes" can be used if the cluster is configured to use external volumes.
+    - "mesos_healthchecks" can be used if ti enable the Mesos HealthChecks (Mesos 1.1 or later).
```

Comment:
I `git blame` this file and it looks like it's Jekyll implementation
(escape underscore. Otherwise jekyl will do a <em> around it)[https://github.com/mesosphere/marathon/commit/6462bb992c43bea1866afcd95f44d0c59c673d34]